docker/hub-agent.Dockerfile

```diff
@@ -14,14 +17,13 @@ COPY cmd/hubagent/  cmd/hubagent/
 COPY apis/ apis/
 COPY pkg/ pkg/
 
-ARG TARGETARCH
-
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} GO111MODULE=on go build -o hubagent  cmd/hubagent/main.go
+RUN echo "Building images with GOOS=$GOOS GOARCH=$GOARCH"
+RUN CGO_ENABLED=1 GOOS=$GOOS GOARCH=$GOARCH GOEXPERIMENT=systemcrypto GO111MODULE=on go build -o hubagent  cmd/hubagent/main.go
```

Comment:
Note: CGO_ENABLED flag has been raisen already on the azure/fleet repo -> the change here simply mirrors that.